tests/soak-tests/src/test/java/org/apache/activemq/artemis/tests/soak/brokerConnection/mirror/LogAssert.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.activemq.artemis.tests.soak.brokerConnection.mirror;
+
+import java.io.File;
+
+import org.apache.activemq.artemis.utils.FileUtil;
+
+public class LogAssert {
+
+   public static void assertServerLogsForMirror(File serverLocation) throws Exception {
+      File log = new File(serverLocation, "log/artemis.log");
+      FileUtil.find(log, l -> l.contains("NullPointerException") || l.contains("AMQ111010"));
```

Comment:
This assert method seems like maybe it isnt asserting anything; it ignores the boolean return value from FindUtil.find() ?